annotations/views/project_views.py

```diff
@@ -179,3 +195,35 @@ def list_projects(request):
         'projects': qs,
     }
     return render(request, template, context)
+
+
+@login_required
+def import_appellation(request, project_id):
+    context = {}
+    text_collection = TextCollection.objects.get(id = project_id)
+    if request.method == 'POST' and request.user == text_collection.ownedBy:
+        data = request.FILES['csv_file']
+        # read csv file
+        csv_file = csv_unicode.reader(data)
+        # Create Temp File.
+        name = 'tmp/'+ request.user.username + '_' + str(datetime.datetime.now())+'.csv'
+        tmp_file = default_storage.save(name, ContentFile(data.read()))
+        # write csv contents to temp file
+        with open(tmp_file, 'w') as f:
+            writer = csv.writer(f)
+            for row in csv_file:
+                try:
+                    writer.writerow(row)
+                except Exception as e:
+                    print ('Error in writing row:',e)
```

Comment:
presumably, if one row can't be written, all of them can't. or at least it'll result in an incomplete file. Either way, the user should know about it.